### DISCUSSION TITLE
gtsam Matlab wrapper

### DISCUSSION THREAD

**Discussion by Amjad-58** [borglab/gtsam#1653] [General]
I am using gtsam Matlab wrapper but I can not find graph.remove, graph.at, graph.atArray and push_back and so on.

**Comment by varunagrawal** [comment]
Hi @Amjad-58. I'm unable to reproduce your issue. Did the Matlab wrapper compile correctly? Are you using the latest develop branch?

Please give us more information about your system, approach and any other relevant details.
